multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
Ok, Google Playstore use a single detail page template for different multimedia and it still looks very good. So let's keep a unified template even for the detail page. 

@nathanajah please push your latest book detail page, so that @digawp won't have too much conflict on the design you already made.

@digawp Separating apps usually does not limit the capability of the templates (you can still have a unified templates with different apps name), it is just a matter of organizing the code when it grows later. However, since our code is still small, we will separate them later when we need to.